src/style.css

```
@@ -1,3 +1,56 @@
-div {
-  background-color: aqua;
+:root {
+  --palette-primary: #ff4900;
+  --palette-grey: #ebebeb;
+  --max-width: 1152px;
```

Use proper naming functions so that, the name itself explains the usage
```suggestion
  --container-max-width: 1152px;
```